### DISCUSSION TITLE
Update Mermaid to allow mindmaps and timeline

### DISCUSSION THREAD

**Discussion by Rufmord** [outline/outline#5214] [Ideas]
Please update the current mermaid version `9.3.0` to `10.x.x`. This will allow [mindmaps ](https://mermaid.js.org/syntax/mindmap.html) and [timelines](https://mermaid.js.org/syntax/timeline.html).

**Comment by tommoor** [comment]
See https://github.com/outline/outline/pull/5168 – unfortunately there are a lot of changes between these two versions that make it incompatible so it will increase the time to upgrade

**Reply by klinshy** [reply]
any updates on this already?

Alternatively, markdown mindmap could be nice to have

**Reply by tommoor** [reply]
`v0.82.0` includes the latest version of Mermaid, this issue is 2 years old

**Reply by klinshy** [reply]
ok thanks
